src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+</head>
+<body>
+    <section class="to-do">
+      <ul class="to-do-list">
+        <li class="list-head">
+            <h3>Today's To Do</h3>
+            <i class="fa-solid fa-arrows-rotate"></i>
+        </li>
+        <li>
+          <input type="text" id="add-list" placeholder="Add to your list">
+          <i class="fa-solid fa-right-to-bracket add-btn" id="add-btn"></i>        
+        </li>
+        <li class="clear-completed"><span class="clear-all-completed">Clear all completed</span></li>
+      </ul>
+
+    </section>
```

Comment:
- `[OPTIONAL]`: Please follow the correct `HTML` semantics. Always try to put the `main` tag to represent the `main` code of the website. 
- Also I would friendly suggest you as an option, have at least one heading per `section` to overcome the `accessibility` issue.